plugins/inputs/github/README.md

```diff
@@ -0,0 +1,45 @@
+# GitHub Input Plugin
+
+The [GitHub](https://www.github.com) input plugin gathers statistics from GitHub repositories.
+
+### Configuration:
+
+```toml
+[[inputs.github]]
+    repositories = [
+      "owner/repository",
```

Comment:
`influxdata/telegraf` would be a good default.  Super nit: just do it as a single line `repositories = ["influxdata/telegraf"]`